README.md

```diff
@@ -6,4 +6,29 @@
 [![Codecov](https://img.shields.io/codecov/c/github/baloise/corellia.svg)](https://codecov.io/gh/baloise/corellia)
 [![gitpod-IDE](https://img.shields.io/badge/open--IDE-as--gitpod-blue.svg?style=flat&label=openIDE)](https://gitpod.io#https://github.com/baloise/corellia)
 
+## about
+
+The project hosts a classical b2b-interface (REST) specification for (insurance) contract data and document exchange.
```

Comment:
Probably should be called "Web-Api" instead of REST

Reply:
accepted